--- boxes-2.3.1.orig/src/Makefile	2026-05-20 17:35:37.012161746 +0200
+++ boxes-2.3.1/src/Makefile	2026-05-20 17:35:37.092038010 +0200
@@ -65,15 +65,12 @@
 
 boxes: $(ALL_OBJ) | check_dir
 	$(CC) $(LDFLAGS) $^ -o $@ -lunistring -lpcre2-32 -lncurses
-	if [ "$(STRIP)" = "true" ] ; then strip $@ ; fi
 
 boxes.static: $(ALL_OBJ) | check_dir
 	$(CC) $(LDFLAGS) $^ -o $(BOXES_EXECUTABLE_NAME) -l:libunistring.a -l:libpcre2-32.a -l:libncurses.a
-	if [ "$(STRIP)" = "true" ] ; then strip $(BOXES_EXECUTABLE_NAME) ; fi
 
 boxes.exe: $(ALL_OBJ) | check_dir
 	$(CC) $(LDFLAGS) $^ -o $@ -lkernel32 -l:libunistring.a -l:libpcre2-32.a -l:libiconv.a
-	if [ "$(STRIP)" = "true" ] ; then strip $@ ; fi
 
 
 flags_unix:
