diff -urN boxes-1.1.orig/src/Makefile boxes-1.1/src/Makefile
--- boxes-1.1.orig/src/Makefile	2000-03-18 00:53:59.000000000 +0100
+++ boxes-1.1/src/Makefile	2003-08-10 12:56:00.000000000 +0200
@@ -99,10 +99,10 @@
 
 
 build:
-	$(MAKE) CFLAGS_ADDTL=-O boxes
+	$(MAKE) boxes
 	strip boxes
 debug:
-	$(MAKE) CFLAGS_ADDTL=-g boxes
+	$(MAKE) boxes
 
 boxes: $(ALL_OBJ)
 	$(MAKE) -C regexp CC=$(CC) libregexp.a
